packages/babel-helpers/src/helpers.ts

```diff
@@ -1932,3 +1932,10 @@ helpers.identity = helper("7.17.0")`
     return x;
   }
 `;
+
+helpers.setFunctionName = helper("7.20.1")`
+  export default function _setFunctionName(f, n) {
```

Comment:
To mirror https://tc39.es/ecma262/#sec-setfunctionname, the helper should support `prefix` used by getters / setters, though it is currently not used by this plugin.